src/components/visualizations/chart/Chart.tsx

```diff
@@ -15,7 +16,9 @@ drillmodule(Highcharts);
 treemapModule(Highcharts);
 funnelModule(Highcharts);
 heatmap(Highcharts);
+HighchartsMore(Highcharts);
 initChartPlugins(Highcharts);
+funnelModule(Highcharts);
```

Comment:
This is redundant, already present above, perhaps introduced in rebase (https://github.com/pbenes/gooddata-react-components/pull/11/files#diff-b4f879edcf6e56d2c3934ffdc4147315R17)